src/Type/InterfaceType/NodeWithFeaturedImage.php

```diff
@@ -24,7 +24,7 @@ public static function register_type( TypeRegistry $type_registry ) {
 			'NodeWithFeaturedImage',
 			[
 				'description' => __( 'A node that can have a featured image set', 'wp-graphql' ),
-				'interfaces'  => [ 'Node', 'DatabaseIdentifier' ],
+				'interfaces'  => [ 'Node' ],
```

Comment:
Still think we should remove `Node` as well for parity with the other `NodeWith` interfaces, and an `exclude` API with the fewest possible quirks. (Why should removing `NodeWithFeaturedImage` be required to remove `Node` and not either remove_all the_ `NodeWith*` interfaces, or without removing _any_ other interfaces. )

If this is an intentional decision, and not something that fell through the cracks of slack vs github, feel free to ignore - though we might consider opening an issue to standardize this behavior and _add_ `Node` to the other `NodeWith*` interfaces.

Reply:
Actually, I think we should go the opposite way. 

Any of the `NodeWith*` Interfaces _should_ be guaranteed to be a Node, so they should _all_ implement the `Node` interface. 

I think the oversight was the other way. 

All `NodeWith *` are nodes, and the Schema _should_ express that. 

Documentation should be clear that excluding connections will only exclude connections implemented directly on that type. If the connections are inherited from an Interface, they would need to be excluded/filtered out from that interface (thus affecting all other Types that implement that interface as well).

Reply:
Like what I was running into the other day was just me _assuming_ this would work differently. The system was doing what it should be doing, preventing me from doing something silly.

Reply:
Issue created: https://github.com/wp-graphql/wp-graphql/issues/2536